src/module_ncio.f90

```diff
@@ -5,52 +5,77 @@
 !! variables and char variables. Variables can have up to 5 dimensions.
 !! @author jeff whitaker <jeffrey.s.whitaker@noaa.gov> @date 201910
 module module_ncio
```

Comment:
At the beginning of the file we need two doxygen docblocks, one for the file, one for the module:

```
!> @file 
!! @brief Brief description of the file.
!! @author Freddy Fortran

!> This module does some important things.
!!
!! @author Freddy Fortran
module mod1
```